lib/sdp/simple_search.rb

```diff
@@ -0,0 +1,61 @@
+module SDP
+  module SimpleSearch
+    def self.search(type, query_string, current_user_id = nil, limit = 20, page = 1)
+      types = type ? [type_to_class(type)] : [Form, Question, ResponseSet]
```

Comment:
I was first, ternary operator stays! Besides, what if type isn't one of the 3 expected Types?!